index.md

```diff
@@ -1229,11 +1248,23 @@ s = "{0} and {1}".format("a", "b")
 
 ### Anomalous backslash in string (W1401) {#W1401}
 
-This error occurs when a backslash is in a literal string but not as an escape.
+This error occurs when a string literal contains a backslash which is not part of an escape sequence.
 
 ~~~~ {include="W1401_anomalous_backslash_in_string"}
 ~~~~
 
+In order to make it explicit that the backslash character represents a backslash, you should escape the backslash with another backslash, or create a *raw string* by prefixing the string literal with the letter `r`.
+
+```python
+re.findall('(\\d+)\\D*', '123 Main St.')  # OK
```

Comment:
So you should change this example to just show some basic valid escape sequences.